src/citrine/resources/material_run.py

```diff
@@ -109,9 +109,9 @@ def get_type(cls) -> Type[MaterialRun]:
         """Return the resource type in the collection."""
         return MaterialRun
 
-    def get_history(self, scope, id) -> Type[MaterialRun]:
+    def get_history(self, scope: str, id: Union[str, UUID]) -> Type[MaterialRun]:
```

Comment:
I don't know if we want to go down this path here, but I think best practice would be to change the signature to:
```
    def get_history(self, id: Union[str, UUID, LinkByUID]) -> Type[MaterialRun]:
```
or
```
    def get_history(self, id: Union[str, UUID, LinkByUID, MaterialRun]) -> Type[MaterialRun]:
```
At the least, it violates policy around positional arguments in that it accepts `runs.get_history(str, str)` where the ordering of those arguments is not obvious.  It also does not support `runs.get_history(CitrineID)`, which would be an obvious first thing to try.  Elsewhere, when giving a scope:ID pair, we have used LinkByUID to avoid this.

Reply:
I think that it would be a good change. But it's a breaking change, which I'm avoiding for the time being.

We could roll it up into https://citrine.atlassian.net/browse/PLA-6584